chemprop/v2/tests/unit/utils/test_converter.py

```diff
@@ -0,0 +1,20 @@
+import os
+import pytest
+from pathlib import Path
+
+from chemprop.v2.utils.v1_to_v2 import convert_model_file_v1_to_v2
+from chemprop.v2.models.model import MPNN
+
+TEST_DIR = Path(__file__).parents[2]
+TEST_DATA_DIR = TEST_DIR / "data"
+
+
+@pytest.fixture
+def example_model_v1_path():
+    return TEST_DATA_DIR / "example_model_v1.pt"
+
+def test_converter(example_model_v1_path):
+    convert_model_file_v1_to_v2(example_model_v1_path, "example_model_v2.ckpt")
```

Comment:
use the pytest `tmp_path` [fixture](https://docs.pytest.org/en/6.2.x/tmpdir.html) here so that you don't actually need to clean up the written checkpoint. The reason this matters is that this test, as written, could fail for file writing/removal, which is outside the scope of the test.